BiometricKit/BiometricKitEnrollProgressCoordinates.h

```diff
@@ -0,0 +1,8 @@
+API_AVAILABLE(ios(7.0))
+@interface BiometricKitEnrollProgressCoordinates : NSObject
+
+@property (assign, nonatomic) double x;
+@property (assign, nonatomic) double y;
+@property (assign, nonatomic) double angle;
```

Comment:
Maybe these are `CGFloat`?

Reply:
I believe you're correct. Should I import `UIKit` into the file?

Reply:
CGFloat is in CoreGraphics, so no need to bring UIKit into this. If these are truly CGFloats and not just doubles

Reply:
Adding this comment primarily for documentation: I launched a 32-bit process on an iOS 10 device, and dumped `UIView` to verify that `CGFloat`s were being dumped as `float`s. I attempted to load BiometricKit and was met with:

```txt
dlopen(/System/Library/PrivateFrameworks/BiometricKit.framework/BiometricKit, 0): no suitable image found.  Did find:
    /System/Library/PrivateFrameworks/BiometricKit.framework/BiometricKit: mach-o, but wrong architecture
```

(most likely this means that BiometricKit is not shipped with a 32-bit slice)

I have decided to change these types to `CGFloat`s and import `CoreGraphics` because these values look like they may be passed into the `CGAffineTransform` API (not verified)